### ISSUE TITLE
Links cannot be distinguished from other text content

### ISSUE THREAD

**Issue by sdellis** [pulibrary/pul_library_drupal#1319]
See this page for example: https://library.princeton.edu/help

There are help links in the content blocks. There are also links in the side menus that aren't apparent. Even though you can hover to get a line beneath them, anyone who's using mobile/tablet will definitely miss them.

**Comment by kevinreiss** [comment]
Still a major issue flagged by key sites audit WCAG Violation

```3.2.4 AA: Components that have the same function are identified consistently.  ```

**Comment by carolyncole** [comment]
The default is now to have underlines
